guide/interactions/slash-command-permissions.md

```diff
@@ -1,104 +1,221 @@
 # Slash command permissions
 
-Slash commands have their own permissions system, which allows you to control who has access to use which commands. Unlike the slash commands permission setting within the Discord client, you can fine-tune access to commands without preventing the selected user or role from using all commands.
+Need to keep some of your commands safe from prying eyes, or only available to the right people? Slash commands support permission overwrites! For both guild and global commands, you can enable or disable a specific **user** or **role** in a guild from using a command.
```

Comment:
```suggestion
Slash commands have their own permissions system, which allows you to control who has access to use which commands. Unlike the slash commands permission setting within the Discord client, you can fine-tune access to commands without preventing the selected user or role from using all commands.
```

Reply:
> Unlike the slash commands permission setting within the Discord client

which permission setting? is it referring to the defaultPermission? Also, I believe this is outdated because it says "with the Discord client". The old revision of the guide used the main library to register commands with a `!deploy` in `messageCreate` event.

I would suggest:

> Slash commands have their own permissions system, which allows you to control which user or role has access to use specific commands.

Reply:
May I suggest:

> Slash commands have their own permission system, which allows you to control who has access to use which commands. You can enable or disable a specific user or role from using a specific command.

Reply:
Seems good to me